### PR TITLE
[Task #2970] Add support multiple microservice yml(#2970)

### PR DESCRIPTION
**Pull request by lishuo5263**
// Describe your PR here; eg. Fixes #issueNo

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-examples/shenyu-examples-springcloud/pom.xml [hunk 1]
```diff
@@ -53,11 +53,11 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
 
- <!--       <dependency>
+        <dependency>
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-springcloud/pom.xml:56]: used spring profiles active nacos , then dependency jar

File: shenyu-examples/shenyu-examples-springcloud/pom.xml [hunk 2]
```diff
@@ -70,11 +70,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
-            <version>${spring-cloud.version}</version>
-        </dependency>
+<!--        <dependency>-->
```
Comment (yu199195) [inline shenyu-examples/shenyu-examples-springcloud/pom.xml:73]: used spring profiles active eureka , then dependency jar

File: shenyu-examples/shenyu-examples-springcloud/pom.xml (hunk 1 above)
Comment (lishuo5263) [reply shenyu-examples/shenyu-examples-springcloud/pom.xml:56]: Yeah, now bootstrap avtive is nacos profiles,so use this.

File: shenyu-examples/shenyu-examples-springcloud/pom.xml (hunk 2 above)
Comment (lishuo5263) [reply shenyu-examples/shenyu-examples-springcloud/pom.xml:73]: I create application-eureka.yml,and activited nacos profiles , so  annotation this eureka pom

File: shenyu-examples/shenyu-examples-springcloud/pom.xml [hunk 3]
```diff
@@ -91,6 +82,32 @@
...
+                    <artifactId>spring-cloud-starter-alibaba-nacos-discovery</artifactId>
+                    <version>${nacos-discovery.version}</version>
+                </dependency>
+            </dependencies>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+        </profile>
+        <profile>
+            <id>eureka</id>
```
Comment (lxl910128) [inline shenyu-examples/shenyu-examples-springcloud/pom.xml:99]: Have you modified the .github/workflows/integrated-test.yml?
because you add two profiles

Comment (lishuo5263) [reply shenyu-examples/shenyu-examples-springcloud/pom.xml:99]: No, I didnt noticed this yaml, thanks of remind .

File: .github/actions/skip-ci/action.yml [hunk 4]
```diff
@@ -67,6 +67,7 @@ runs:
         done
 
         echo "SKIP_CI=true" >> $GITHUB_ENV
+        echo "EXAMPLE_SPRINGCLOUD_FLAG=true" >> $GITHUB_ENV
```
Comment (yu199195) [inline .github/actions/skip-ci/action.yml:70]: why add this？

File: .github/workflows/integrated-test.yml [hunk 5]
```diff
@@ -61,6 +61,9 @@ jobs:
       - name: Build examples
         if: env.SKIP_CI != 'true'
         run: ./mvnw -B clean install -Pexample -Dmaven.javadoc.skip=true -Dmaven.test.skip=true -f ./shenyu-examples/pom.xml
+      - name: Build example-springcloud
```
Comment (yu199195) [inline .github/workflows/integrated-test.yml:64]: why add this

File: .github/workflows/integrated-test.yml [hunk 6]
```diff
@@ -61,6 +61,9 @@ jobs:
...
+        if: env.EXAMPLE_SPRINGCLOUD_FLAG = 'true'
+        run: ./mvnw -B clean install -Peureka -Dmaven.javadoc.skip=true -Dmaven.test.skip=true -f ./shenyu-examples/shenyu-examples-springcloud/pom.xml
       - name: Build integrated tests
```
Comment (yu199195) [inline .github/workflows/integrated-test.yml:67]: why remove Build integrated tests？

**Review by yu199195 — APPROVED** [review]

**Pull request merged**